README.md

```diff
@@ -320,28 +293,11 @@ the plugin verifies that the pre deployment checks have terminated
 by retrieving the current status of the WorkloadInstance.
 Only when that is successful is the pod bound to a node.
 
-## Install a dev build
+## Contributing
 
-The [GitHub CLI](https://cli.github.com/) can be used to download the manifests of the latest CI build.
-
-```bash
-gh run list --repo keptn/lifecycle-toolkit # find the id of a run
-gh run download 3152895000 --repo keptn/lifecycle-toolkit # download the artifacts
-kubectl apply -f ./keptn-lifecycle-operator-manifest/release.yaml # install the lifecycle-operator
-kubectl apply -f ./scheduler-manifest/release.yaml # install the scheduler
-```
-
-Instead, if you want to build and deploy the operator into your cluster
-directly from the code, you can type:
-
-```bash
-RELEASE_REGISTRY=<YOUR_DOCKER_REGISTRY>
-# (optional)ARCH=<amd64(default)|arm64v8>
-# (optional)CHART_APPVERSION=<YOUR_PREFERRED_TAG (defaulting to current time)>
```

Comment:
pls add this info to the docs page on how to run local builds in the [contrib guide](https://keptn.sh/stable/docs/contribute/software/dev-environ/#first-steps).

Reply:
@odubajDT with this i meant that some info about the `ARCH` and `CHART_APPVERSION` args is missing at the link i provided 😄